migrations/sql/V2020.03.18.10.40__create_bond_documents_type.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE bond_document_type(
+    bond_document_type_code          varchar                                NOT NULL PRIMARY KEY,
+    description                      varchar                                NOT NULL            ,
+    active_ind                       boolean                  DEFAULT true  NOT NULL            ,
+    create_user                      varchar                                NOT NULL            ,
+    create_timestamp                 timestamp with time zone DEFAULT now() NOT NULL            ,
+    update_user                      varchar                                NOT NULL            ,
+    update_timestamp                 timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE bond_document_type OWNER TO mds;
+COMMENT ON TABLE bond_document_type IS 'Contains a list of documents related to bonds used used by the Ministry. Bond document types include; Scan of Reclamation Security Bond, Release of Security Form (signed by inspector), Release of Security Letter (my letter to the bank or proponent), Confiscation of Security Form (signed by inspector), Confiscation of Security Letter (my letter to bank or proponent), Reminder Letter, Acknowledgement Letter.';
```

Comment:
This comment will go out of date, can we remove the content, as that is in the rows. Show the intent and related tables.

Reply:
also `used used` on the first line.

Reply:
Is it possible to leave a couple examples of a bond type in the description, it helps provide context to someone that may not have access to the database. Maybe just update the comment to say something like "Examples of bond types may include... and then have 2 or 3"?

Reply:
I have changed this based on the suggestions to be a may include with 3 examples